addons/popochiu/editor/config/config.gd

```diff
@@ -25,10 +36,17 @@ func initialize_editor_settings():
 
 
 func initialize_project_settings():
+	_initialize_project_cfg(_DEFAULT_INVENTORY_ALWAYS_VISIBLE, false, TYPE_BOOL)
+	_initialize_project_cfg(_DEFAULT_SCALE_GUI, true, TYPE_BOOL)
```

Comment:
Defining the setting is straightforward, but I am not yet sure how we use it in the codebase. Searching for `scale_gui` for example, only shows that is defined in `popochiu_settings.tres` but I don't see where that value is read.

Reply:
@stickgrinder can you help me understand how these settings should be used in the project? For example, I define `is_default_scale_gui()` method below to access the setting, but I am not sure where to call this method.

In the settings resource file I see `scale_gui = true`, but not sure where `scale_gui` is then used.

Reply:
I think I figured this out with https://github.com/mapedorr/popochiu/pull/66/commits/212a85b93d6dd88de51bab6a3024787894e5531a, instantiating `projectSettings` in `popochiu.gd` and using is there. I can move the rest of the settings over now, but @stickgrinder can you take a look at this latest commit and let me know if I'm on the right track?